operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -24,18 +22,18 @@ description: 50% Draft
 * Release Manager leads the team to arrive at a decision on the theme / main highlight for the blog post. Consult with marketing and PM leads for final approval
   * Take into consideration whether a feature is highly requested, what is the overall impact of all marketed features, and whether a feature is at risk of being cut
 
-### B. \(T-minus 8 working days\) First Draft
```

Comment:
One consideration that comes to mind: looking at this release cycle, we had the first draft at T-11, PM review at T-9 and head of PM review at T-6.

Do you think that was a good cadence, or any dates that should move back / forward compared to this release cycle, to meet the T-5 deadline?